### PR TITLE
Integrate independently buildable PoseDaemon [ESD-1039]

### PR DESCRIPTION
**Pull request by denniszollo**

### CODE REVIEW

File: package/piksi_ins/piksi_ins.mk [hunk 1]
```diff
@@ -17,13 +17,14 @@ endif # ! ($(BR2_BUILD_RELEASE_PROTECTED),y)
 
 $(info >>> Piksi INS is enabled, packaging with current image)
 
-PIKSI_INS_VERSION = 4e3c77a484bd4d3f0f53c4ab459b6966fc31c541
-PIKSI_INS_SITE = git@github.com:carnegieroboticsllc/piksi_ins.git
+PIKSI_INS_VERSION = master
```
Comment (silverjam) [inline package/piksi_ins/piksi_ins.mk]: This should be tagged to a specific commit hash so master tags are idempotent (that is, not a moving target when rebuilding).

Comment (denniszollo) [reply package/piksi_ins/piksi_ins.mk]: agreed! I'll update when it works.

Comment (silverjam) [reply package/piksi_ins/piksi_ins.mk]: I just updated to pin it, sorry, I'll stop poking at this now.

**Comment by anonymous-travis-swiftnav** [comment]
## HITL smoke tests: v2.1.0-develop-2019011922-5-g078c163c

These test runs are kicked off whenever you push a new commit to this PR. All passfail metrics in these runs must pass for the `hitl/pass-fail` status to be marked successful.

### job status
+ [live-roof-650-townsend-post runs](https://gnss-analysis.swiftnav.com/jobs/id=869111)

### gnss-analysis results
At least one run must complete for these links to have data.

#### passfail"
+ [live-roof-650-townsend-post](https://gnss-analysis.swiftnav.com/metrics_preset=pass_fail&scenario=live-roof-650-townsend-post&summary_type=q50&groupby_key=firmware&firmware_versions=v2.1.0-develop-2019011922-5-g078c163c&build_type=buildroot_pr&display_type=table)

#### detailed
+ [live-roof-650-townsend-post](https://gnss-analysis.swiftnav.com/metrics_preset=detailed&scenario=live-roof-650-townsend-post&summary_type=q50&groupby_key=firmware&firmware_versions=v2.1.0-develop-2019011922-5-g078c163c&build_type=buildroot_pr&display_type=table)

**Comment by anonymous-travis-swiftnav** [comment]
## v2.1.0-develop-2019011922-5-g078c163c
+ [Artifacts (HITL Dashboard Page)](https://gnss-analysis.swiftnav.com/artifacts/build_type=buildroot_pr&name_filter=v2.1.0-develop-2019011922-5-g078c163c)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v2.1.0-develop-2019011922-5-g078c163c/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v2.1.0-develop-2019011922-5-g078c163c

**Comment by anonymous-travis-swiftnav** [comment]
## HITL smoke tests: v2.1.0-develop-2019011922-5-g078c163c

These test runs are kicked off whenever you push a new commit to this PR. All passfail metrics in these runs must pass for the `hitl/pass-fail` status to be marked successful.

### job status
+ [live-roof-650-townsend-post runs](https://gnss-analysis.swiftnav.com/jobs/id=869114)

### gnss-analysis results
At least one run must complete for these links to have data.

#### passfail"
+ [live-roof-650-townsend-post](https://gnss-analysis.swiftnav.com/metrics_preset=pass_fail&scenario=live-roof-650-townsend-post&summary_type=q50&groupby_key=firmware&firmware_versions=v2.1.0-develop-2019011922-5-g078c163c&build_type=buildroot_pr&display_type=table)

#### detailed
+ [live-roof-650-townsend-post](https://gnss-analysis.swiftnav.com/metrics_preset=detailed&scenario=live-roof-650-townsend-post&summary_type=q50&groupby_key=firmware&firmware_versions=v2.1.0-develop-2019011922-5-g078c163c&build_type=buildroot_pr&display_type=table)

**Comment by anonymous-travis-swiftnav** [comment]
## v2.1.0-develop-2019011922-5-g078c163c
+ [Artifacts (HITL Dashboard Page)](https://gnss-analysis.swiftnav.com/artifacts/build_type=buildroot_pr&name_filter=v2.1.0-develop-2019011922-5-g078c163c)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v2.1.0-develop-2019011922-5-g078c163c/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v2.1.0-develop-2019011922-5-g078c163c

**Comment by anonymous-travis-swiftnav** [comment]
## HITL smoke tests: v2.1.0-develop-2019011922-6-g9ff58324

These test runs are kicked off whenever you push a new commit to this PR. All passfail metrics in these runs must pass for the `hitl/pass-fail` status to be marked successful.

### job status
+ [live-roof-650-townsend-post runs](https://gnss-analysis.swiftnav.com/jobs/id=869190)

### gnss-analysis results
At least one run must complete for these links to have data.

#### passfail"
+ [live-roof-650-townsend-post](https://gnss-analysis.swiftnav.com/metrics_preset=pass_fail&scenario=live-roof-650-townsend-post&summary_type=q50&groupby_key=firmware&firmware_versions=v2.1.0-develop-2019011922-6-g9ff58324&build_type=buildroot_pr&display_type=table)

#### detailed
+ [live-roof-650-townsend-post](https://gnss-analysis.swiftnav.com/metrics_preset=detailed&scenario=live-roof-650-townsend-post&summary_type=q50&groupby_key=firmware&firmware_versions=v2.1.0-develop-2019011922-6-g9ff58324&build_type=buildroot_pr&display_type=table)

**Comment by anonymous-travis-swiftnav** [comment]
## v2.1.0-develop-2019011922-6-g9ff58324
+ [Artifacts (HITL Dashboard Page)](https://gnss-analysis.swiftnav.com/artifacts/build_type=buildroot_pr&name_filter=v2.1.0-develop-2019011922-6-g9ff58324)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v2.1.0-develop-2019011922-6-g9ff58324/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v2.1.0-develop-2019011922-6-g9ff58324

**Comment by silverjam** [comment]
@denniszollo Bench test and merge?

**Comment by denniszollo** [comment]
Bench tested and looks good.  I'm merging.  Thanks @silverjam for getting it over the finishline!

**Pull request merged**